airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGeneratorIntegrationTest.java

```diff
@@ -652,18 +678,10 @@ public void testCdcUpdate() throws InterruptedException {
     final String sql = GENERATOR.updateTable("", cdcStreamConfig());
     logAndExecute(sql);
 
-    // TODO
     final long finalRows = bq.query(QueryJobConfiguration.newBuilder("SELECT * FROM " + streamId.finalTableId("", QUOTE)).build()).getTotalRows();
-    assertEquals(4, finalRows);
+    assertEquals(5, finalRows);
```

Comment:
@edgao I think this should be 5 - all 6 users are in the latest batch.  1 of them has non-null _ab_cdc_deleted_at (#1).  "_ab_cdc_deleted_at": {} is null I guess?

Reply:
+1

yeah, deleted_at is supposed to be a timestamp so `{}` turns into a null. previous behavior was just extracting the raw json value and checking for nullness, so previously it was handling this as a delete.

I feel ok with changing this behavior to "do cdc deletes if `_ab_cdc_deleted_at` is a _valid_ non-null value"  - there's no legacy precedent here, since legacy normalization would have just failed entirely.